### PR TITLE
Extend liveness probe with Ceph connection check

### PR DESCRIPTION
**Pull request by fridex**

### CODE REVIEW

File: thoth_user_api/entrypoint.py [hunk 1]
```diff
@@ -72,9 +72,15 @@ def api_readiness():
 @app.route('/liveness')
 def api_liveness():
     """Report liveness for OpenShift readiness probe."""
+    # Check response from Kubernetes API.
     response = requests.get(Configuration.KUBERNETES_API_URL,
                             verify=Configuration.KUBERNETES_VERIFY_TLS)
     response.raise_for_status()
+
+    # Check that Ceph is reachable.
```
Comment (goern) [inline thoth_user_api/entrypoint.py]: shouldnt this be also part of readiness?

**Comment by fridex** [comment]
Fixes: #109

**Comment by fridex** [comment]
Fixes: #109

**Comment by goern** [comment]
hmm, ok, so we dont expose the error reason to the calling client.

/approve

**Pull request merged**